blog/crsp-v2-update/index.qmd

```diff
@@ -0,0 +1,133 @@
+image-alt: A split-screen illustration that shows old, worn-out, and faded financial papers, charts, and documents, characterized by a tattered appearance, yellowed by time, and cluttered arrangement, symbolizing traditional and outdated methods of financial management and analysis on the left side This contrast highlights the evolution of financial documentation towards more modern practices on the right side. Created with DALL-E.
+categories: 
+  - Data
+  - R
+  - Python
+--- 
+
+With commit [6acb50b](https://github.com/tidy-finance/website/commit/6acb50b4d7ee8415be41731bdd65fc88b1bff76b), Tidy Finance has transitioned to a new version of the CRSP tables distributed by WRDS. In this blog post, we will review the changes that affect our routines.
+
+## Overview: CRSP and WRDS
+
+Tidy Finance shows how to download the most used data in empirical research in finance from the [Wharton Research Data Services (WRDS)](https://wrds-www.wharton.upenn.edu/). A main component of this data is the stock return history provided by the [Center for Research in Security Prices (CRSP)](https://crsp.org/). CRSP is the de-facto gold standard for stock return data covering historical returns from the 1920s until the present.
+
+You can read all about how to connect to WRDS and download the CRSP data in our chapter **WRDS, CRSP, and Compustat** in the [R version](../../r/wrds-crsp-and-compustat.qmd) and the [Python version](../../python/wrds-crsp-and-compustat.qmd).
+
+## CRSP format 2.0 (CIZ)
+
+In 2022, CRSP rolled out the new format of its data which is called **CRSP's Stock and Indexes Flat File Format 2.0 (CIZ)**. This new format is now also available via WRDS. So, what is new in the second version? The most notable items from CRSP's announcement includes:
+
+- They renamed many variables in the daily and monthly data. Not all of these changes make it easier to understand what the variables are, and some names are hard to read. 
+- They added additional variables that are sometimes redundant but make querying simpler (e.g., you do not have to merge different data to perform common tasks).
+- Delisting returns are now included in the main return time series.
+- They introduce issuer level data, which involved ensuring unique observations for each issuer (e.g., industry codes).
+- They reworked some flag items by giving them alphanumeric values instead of the previous numeric codes. Moreover, these codes are now also documented in metadata files to facilitate access.
+
+Many of these changes mean, that you will not be able to simply use the new data with your existing code. Do not worry. We have you covered and will show you how to use the new data in your previous routines easily.
+
+You can access the new data via WRDS using the commands below. For the main data WRDS basically just added a "_v2" postfix. However, you also see that we need fewer tables, as we do not load information on delisting returns separately. 
+
+::: {.panel-tabset group="language"}
```

Comment:
Is this tabset really that useful? The R code does not show how they are joined and the Python query is also incomplete. I'd rather skip it to be honest and only refer to the books (also lower maintenance effort on our end in the future in case we need to fix something).

Reply:
Then I just leave the R code (I think it is nice to see a small break in the text.